types.go

```diff
@@ -16,3 +16,8 @@ var (
 	ErrCacheNotSupported = errors.New("Operation not supported")
 )
 
+//Interface for all onecache store implementations
+type CacheStore interface {
+	Set(key string, data interface{}, expires time.Duration) error
+	Get(key string) ([]byte, error)
```

Comment:
As an aside, i am not too comfortable with this interface as it just leads to usability problems. Requesting the user to convert those bytes into whatever is needed.

A better choice might be to return an `interface{}` or allow an `interface{}` be passed as a second param  (but it must be a pointer or must be settable)